src/components/agents/server/procedures.ts

```
@@ -0,0 +1,20 @@
+    getMany: baseProcedure.query(async () => {
+        const data = await db
+            .select()
+            .from(agents);
+
+        // await new Promise(resolve => setTimeout(resolve, 1000));
+        throw new TRPCError({
+            code: 'INTERNAL_SERVER_ERROR',
+            message: 'An error occurred while fetching agents.'
+        });
+    
+        return data;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)